pom.xml

```diff
@@ -20,7 +20,6 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-mongodb</artifactId>
-			<version>2.1.2.RELEASE</version>
```

Comment:
Just for my own clarity, why was 2.1.2 RELEASE not a good version? What's being used now?

Reply:
There was a build conflict when booting up the application. Now it is using 3.1.0 (according to IntelliJ dependencies list) which is a newer version.